packages/forms/src/elements/address/experianAddressLookupProvider.ts

```diff
@@ -0,0 +1,64 @@
+import { AddressAPIType } from '@tpr/layout/lib/components/cards/common/interfaces';
```

Comment:
this import is making fail the circleci test. `@tpr/layout` is not a dependency of `@tpr/forms`.
we could simply add for now the `AddressAPIType` declaration to the file. Until we have a separate `@tpr/types` package.

Reply:
I've followed your suggestion. I actually wanted to get rid of `AddressAPIType` but I couldn't work out how the lookup is getting the base URL for the Experian address service, so I had to leave it in just to let the magic work! If you can see how that works it could just go away rather than creating @tpr/types.